integrations/opennms-snmp-hardware-inventory-provisioning-adapter/src/test/java/org/opennms/netmgt/provision/SnmpHardwareInventoryProvisioningAdapterIT.java

```diff
@@ -230,4 +234,32 @@ public void testDiscoverSnmpEntities() throws Exception {
         Assert.assertEquals(130, m_entityDao.countAll());
     }
 
+    @Test
+    public void testUpdatedAttributeTypes() throws Exception {
+
+        //checking attribute class ( from hwentityattributetype table) before loading
+        //config file (test-config-03.xml)l
+        HwEntityAttributeType testEntityAttributeType = m_adapter.getHwEntityAttributeTypeDao().findTypeByOid(".1.3.6.1.4.1.637.800.1.1.1.1.1.1.1.7");
```

Comment:
Where was this `.1.3.6.1.4.1.637.800.1.1.1.1.1.1.1.7` getting inserted ?

Reply:
its loaded and inserted from etc/snmp-hardware-inventory-adapter-configuration.xml file

Reply:
Ideally we should not depend on external resources in tests.

Reply:
Created separate config file test-config-03.xml (before test start) and test-config-04.xml (after) , To avoid dependency on external file